sda/cmd/download/swagger_v1.yml

```diff
@@ -0,0 +1,256 @@
+              schema:
+                type: array
+        "401":
+          description: Authentication failure
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /info/dataset:
+    get:
+      description: Returns an array with metadata about the dataset
+      parameters:
+        - in: query
+          name: dataset
+          schema:
+            type: string
+          required: true
+          description: Dataset identifier
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DatasetInfo"
+          description: Successful operation
+        "401":
+          description: Authentication failure
+        "403":
+          description: The user does not have access to the dataset
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /info/dataset/files:
+    get:
+      description: Returns an array with metadata about all files in a dataset
+      parameters:
+        - in: query
+          name: dataset
+          schema:
+            type: string
+          required: true
+          description: Dataset identifier
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/FileInfo"
+          description: Successful operation
+        "401":
+          description: Authentication failure
+        "403":
+          description: The user does not have access to the dataset
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /health/ready:
+    get:
+      description: Returns the status of the application.
+      responses:
+        "200":
+          content:
+            application/json:
+              example:
+                {
+                  "c4ghService": "responding",
+                  "database": "responding",
+                  "oidcServer": "responding",
+                  "storageBackend": "responding",
+                }
+          description: The service is operational
+        "503":
+          description: Unhealthy service
+  /health/live:
+    get:
+      description: Returns 200 as long as the main thread is running
+      responses:
+        "200":
+          description: The service is operational
+components:
+  schemas:
+    DatasetInfo:
+      type: object
+      properties:
+        date:
+          type: string
+          format: date-time
+          example: "2025-02-14T14:51:26.639Z"
+        files:
+          type: integer
+          format: int32
+          example: 1234
+        name:
+          type: string
+          example: aa-dataset-123456-asdfgh
+        size:
+          type: integer
+          format: int64
+          example: 6597069766656
```

Comment:
What is this, the combined sum of all dataset contents (with headers?). I'd be happy to have a description.

Reply:
The only relevant size would be the total size of all encrypted files.

Reply:
With the corresponding header sizes (pre-rerencyption) added?